### ISSUE TITLE
When accepting a variable as an argument in Run Keyword(s), code analysis lists the variable as an "Undefined keyword"

### ISSUE THREAD

**Issue by katjie** [robocorp/robotframework-lsp#468]
**Describe the bug**
If, instead of calling a keyword directly, I use a variable (in my case, defined from variables.py) in Run Keyword(s), then I get the "undefined keyword" error for these variables:

**To Reproduce**
1. Set a variable as a Keyword
2. Write Run Keyword  ${VARIABLE}
3. This will be reported as a problem where ${VARIABLE} is an undefined keyword

**Expected behavior**
If it is an actual keyword which is passed, which is undefined, I expect to see this undefined keyword error.
However, if it is a variable that is passed, I do not expect it to raise an error.

**Screenshots**
![image](https://user-images.githubusercontent.com/19191177/135841573-72384264-d1c2-45de-99df-624efe1b6beb.png)

**Versions:**
 - OS: Windows 10
 - Robot Framework Version 4.1.1, Python 3.9.6 (32-bit)
 - Robot Framework Language Server Version 0.23.2
 - VSCode 1.60.2

**Logs**
[robotframework-lsp.lsp.2132.log](https://github.com/robocorp/robotframework-lsp/files/7277623/robotframework-lsp.lsp.2132.log)
[robotframework-lsp.remote_fs_observer.14260.log](https://github.com/robocorp/robotframework-lsp/files/7277624/robotframework-lsp.remote_fs_observer.14260.log)

**Comment by katjie** [comment]
Hello @fabioz 

I can see that this bug has been fixed and in the example I wrote in my initial issue report, it does indeed no longer give me the "Undefined keyword" error.

However, if I were to use the same approach to call a given keyword with a variable as part of the keyword name:
![image](https://user-images.githubusercontent.com/19191177/148078466-039aa4cf-dffe-487d-9bf6-2ccbeda73c35.png)

Then I am met with the error again. In my final setup, variable ``RUN_INTERFACE`` would be passed when executing robot framework, and would then change whether I want to run ``A Start Running`` or ``B Start Running``.

Is this an extension of this bug, which should be opened again, or shall I make a new bug report for this issue?
